webui/react/src/pages/TrialDetails/Profiles/Charts/SystemMetricChart.tsx

```diff
@@ -90,7 +86,7 @@ const SystemMetricChart: React.FC<ChartProps> = ({ getOptionsForMetrics, trial }
         )
       }
       title="System Metrics">
-      <UPlotChart data={systemMetrics.data} options={options} />
+      <LineChart series={systemMetrics.data} xAxis={XAxisDomain.Time} xLabel="Time" />
```

Comment:
We should add a Y label, `yLabel={getUnitForMetricName( ... )}`  function is in `src/pages/TrialDetails/Profiles/utils.ts`

Also seems to be an issue with width of the chart and axis labels with these larger numbers, here's an example with disk_throughput_read

<img width="605" alt="Screen Shot 2023-01-27 at 12 57 01 PM" src="https://user-images.githubusercontent.com/643918/215172226-d0991e67-18b8-472f-8851-a794f31d7403.png">

Reply:
fixed